server/src/game/client_actor.rs

```diff
@@ -0,0 +1,125 @@
+use std::time::{Duration, Instant};
+
+use actix::prelude::*;
+use actix_web_actors::ws;
+
+use super::messages::{Connect, Disconnect, ServerMessage};
```

Comment:
Would prefer these imports to be absolute (e.g. `crate::`) instead of relative. I think it makes modules easier to relocate later and avoids decisions around whether to use absolute or relative locations for a particular import.